bin/ch-pull2tar

```diff
@@ -20,12 +20,46 @@ EOF
 
 parse_basic_args "$@"
 
-if [ "$#" -ne 2 ]; then
+if [ "$#" -lt 2 ]; then
     usage
 fi
 
 image=$1
 dest=$2
+tar="$image".tar.gz
 
-docker_ pull "$image"
-"${ch_bin}/ch-builder2tar" "$image" "$dest"
+while [ $# -gt 0 ]; do
+    opt=$1; shift
+    case $opt in 
+        -p|--pull-from) 
+            if [ "$1" = "docker" ]; then
+                docker=yes
+            elif [ "$1" = "skopeo" ]; then 
+                skopeo=yes
+            else 
+                echo "$1 is an invalid option" 1>&2
+                exit 1
+            fi
+            ;;
+    esac
+done
+
```

Comment:
Roughly here, add logic to select a default puller if none specified; also, say which puller was selected and why.